src/EventDelegate/index.ts

```diff
@@ -47,16 +48,18 @@ export class EventDelegate {
 
     if (popEngine.isPopForTrigger(target, trigger)) {
       if (popEngine.isPopAlreadyOpenForTarget(target)) {
-        popEngine.clearTimeout(target);
+        let groupId = target.getAttribute('popgun-group');
+        timeoutManager.maybeClearTimeout(timeoutManager.getTimeouts().timeToHoverOnPop, groupId);
```

Comment:
Looking at this, I see why you must cringe on the inside lol. Since these are pretty finite things we are looking for, thoughts on...
`timeoutManager.maybeClearTimeout(TimeoutManager.TIME_TO_HOVER_ON_POP, groupId);` ?? Is that even feasible?